src/js/tabs/osd.js

```diff
@@ -707,6 +707,14 @@ OSD.constants = {
                FONT.symbol(SYM.HEADING_DIVIDED_LINE) + FONT.symbol(SYM.HEADING_LINE) + FONT.symbol(SYM.HEADING_E)
       }
     },
+    FLIP_ARROW: {
```

Comment:
This should be added at the end of the list, below `LOG_STATUS`, and not in the middle.

Also, this will have to be added to `OSd.chooseFields()` (https://github.com/betaflight/betaflight-configurator/blob/eb78683e90385d3e4a6a15d2d2ddd152eae6186d/src/js/tabs/osd.js#L956) for `apiVersion` >= 1.41.0, or else it will not be shown.